operations/validate_csm_health.md

```diff
@@ -66,16 +66,16 @@ There are multiple Goss test suites available that cover a variety of sub-system
 
 Run the NCN health checks with the following command:
 
-**IMPORTANT:** Do not run these as part of upgrade testing. This includes the Kubernetes check in the next block. 
+**IMPORTANT:** Do not run these as part of upgrade testing. This includes the Kubernetes check in the next block. If m001 is the PIT node, run the following on the PIT, otherwise run the healthchecks from any NCN.
```

Comment:
The first sentence says not to run these and this addition says to run it? I think these steps are confusing.

Reply:
fixed - please check to see if this makes more sense